docs/faststore/docs/reference/icons.mdx

```diff
@@ -79,13 +79,19 @@ You can find more details about the `Icon` component [here](https://developers.v
 
 <IconList>
   <IconComponent icon={<Icon name="Visa" />} name="Visa" />
+  <IconComponent icon={<Icon name="Amex" />} name="Amex" />
+  <IconComponent icon={<Icon name="ApplePay" />} name="ApplePay" />
```

Comment:
**issue:** some icons are missing.
**question:** how are we importing those icons?
 
<img width="974" alt="image" src="https://github.com/user-attachments/assets/36b6cd4d-2a40-47ed-ad14-10d112b62b6d">